source/includes/documentation/_material_ui_elements.md

```diff
@@ -2919,39 +2919,44 @@ Available methods in Java JDI Light:
     @UI("span.MuiButton-label")
     public static Menu menu;
 
-    //  @FindBy(css = "ul.MuiList-root.MuiMenu-list.MuiList-padding")
-    @UI("ul.MuiList-root.MuiMenu-list.MuiList-padding")
-    public static List simpleMenuList;
+    //  @FindBy(css = "#selectedItem")
+    @UI("#selectedItem")
+    public static Text selectedSimpleMenuItem;
 
-    //  @FindBy(xpath = "//*/p[starts-with(@class, 'MuiTypography')]")
-    @UI("//*/p[starts-with(@class, 'MuiTypography')]")
-    public static Menu contextMenu;
+    //  @FindBy(css = "p.MuiTypography-root")
+    @UI("p.MuiTypography-root")
+    public static Text pageText;
 
-    //  @FindBy(css = "ul.MuiList-root.MuiMenu-list.MuiList-padding")
-    @UI("ul.MuiList-root.MuiMenu-list.MuiList-padding")
-    public static List contextMenuList;
+    //  @FindBy(css = ".MuiMenuItem-root")
+    @UI(".MuiMenuItem-root")
+    public static Menu contextMenuList;
 
     @Test
     public void simpleMenuTest() {
-        menu.has().text("OPEN MENU");
-        menu.click();
-        simpleMenuList.is().displayed();
-        menu.has().properMenuItems(simpleMenuList, SIMPLE_AND_SELECTED_MENU_ITEMS);
-        simpleMenuList.selectItemByText("Profile");
-        menu.is().displayed();
+        simpleMenuButton.show();
+        simpleMenuButton.has().text("OPEN MENU");
+        simpleMenuButton.click();
+
+        String option = "Profile";
+        menu.select(option);
+        selectedSimpleMenuItem.has().text("Selected menu: " + option);
     }
 
     @Test
     public void contextMenuTest() {
         contextMenuPage.open();
-        contextMenu.rightClick();
-        menu.has().properMenuItems(contextMenuList, CONTEXT_MENU_ITEMS);
-        contextMenuList.selectItemByText("Print");
-        contextMenu.isDisplayed();
+        contextMenuPage.isOpened();
+        pageText.is().displayed();
+
+        pageText.rightClick();
+        menu.is().displayed().and().has().itemsTexts(CONTEXT_MENU_ITEMS);
```

Comment:
preferable to replace it with `Arrays.asList("Copy", "Print", "Highlight", "Email")`